FESS-GUI/flypacketlib.h

```diff
@@ -0,0 +1,8 @@
+#ifndef FLYPACKETLIB_H
+#define FLYPACKETLIB_H
+
+#include "datatypes.h"
+
+flypacket buildFlyPacket(flybyte);
```

Comment:
The idea here is to create a library that can be shared between the gui and the mcu for encoding and decoding commands. This way we don't need to duplicated work. The mcu is written in C hence the struct. If it is possible to use C++ on the mcu I will convert it to a class.